lib/dialects/abstract/query.js

```diff
@@ -324,14 +324,22 @@ class AbstractQuery {
   /**
    * @param {string} sql
    * @param {Function} debugContext
+   * @param {Array|Object} parameters
    * @protected
    * @returns {Function} A function to call after the query was completed.
    */
-  _logQuery(sql, debugContext) {
+  _logQuery(sql, debugContext, parameters) {
     const { connection, options } = this;
     const benchmark = this.sequelize.options.benchmark || options.benchmark;
     const startTime = Date.now();
-    const fmt = `(${connection.uuid || 'default'}): ${sql}`;
+    let logParameter='';
+    if (!sql.endsWith(';')) {
+      logParameter=';';
+    }
+    if (parameters) {
+      logParameter+=JSON.stringify(parameters);
```

Comment:
As I said in comments below, we want one standard style for appending `parameters` after SQL. I think comma separated values works best.

Also please use space before/after `=` :)

Reply:
Thanks a lot. The parameters are object for mssql and sqlite. Should remain them in log ?
like
```js
INSERT INTO `Users` (`id`,`username`,`emailAddress`) VALUES (NULL,$1,$2);{"$1":"john","$2":"john@gmail.com"}
```
and 
```js
INSERT INTO `Users` (`id`,`username`,`emailAddress`) VALUES (NULL,$one,$two);{"$one":"john","$two":"john@gmail.com"}
```

Reply:
No they should be appended as array to final sql. Like `SQL SQL SQL; value1, value2, value3`. You will have to convert object to array in `_logQuery` function

Reply:
@sushantdhiman So the append values have the same order and  repeat with placeholders ?

Reply:
Hmm object property ordering could be an issue, when it is an array append it, when it is an object stringify it.

I think we might want to hide this behind extra flag like `sequelize.options.loggingParameters= false`

Reply:
I have a question. Single quotes in PostgreSQL are used to create a text string, so should we convert string to single quotes for postgreSQL?
eg. 
```js
INSERT INTO "users" ("id","name") VALUES (DEFAULT,$1) RETURNING *; 'john'
```
In my opinion, sometimes the parameters will handle again by Individual module . For example postgres use [pg](https://github.com/brianc/node-postgres) .
So we remain what we send to this module. Just  as
```js
   //_logQuery.js
    if (!sql.endsWith(';')) {
      logParameter=';';
    }
    if (parameters) {
      if (Array.isArray(parameters)) {
        logParameter = logParameter+' '+parameters.map(p=>JSON.stringify(p)).join(', ')
      } else {
        logParameter += JSON.stringify(parameters);
      }
    }
```
Is this ok?

Reply:
These queries are only being logged and there no way you can actually execute what we log `SQL; parameters` is just for our logging purposes. There is no need to quote parameter strings in any particular style when logging then

Reply:
I agree with you .But 
```js
sequelize.query(`select $1 as foo,$2 as bar`, { bind: ['foo, bar','bar'])
```
If we don't quote strings ,It would be 
```
select $1 as foo,$2 as bar; foo, bar, bar
```

Reply:
Right, this is getting complicated (i thought we could just append them). There are our issues so far

1. We don't have same format for all dialects, due to object vs array

Possible Sol: I think bindParam is just an array, we should be able to get array of parameters for all cases. Perhaps check `options.bindParams` if that is available in context.

2. String should be properly quoted, so they don't lose their semantics in relation for bounded variables.

Possible Sol: JSON.stringify seems like a good idea for that

3. There are buffers types, do we want to print them all out. That result in significant noise in logs

Possible Sol: filter object or array for buffer representation or hexified data and omit that before json stringify

Don't need to worry too much about (3) now, but we will tackle that at the end of this PR. Let's start with (1) and (2) first

Reply:
1 and 2 is ok. Please check it. Furthermore, although `options.bindParams` is an array, but sql may contain `$one` etc.

Reply:
No problem, at-least all bounded variables will be logged in same format by using parameters from `options.bindParams`

Reply:
Yeah, i know, i will modify it.

Reply:
`options.bind`?

Reply:
There is not `options.bindParams` , `options.bind` and `options.replacements` may be object.